### DISCUSSION TITLE
How can find out what version of WinUI a maui app is using?

### DISCUSSION THREAD

**Discussion by upswing1** [dotnet/maui#18350] [Q&A]
I am developing a MAUI App running under windows using .net8 preview.  I read today that a new version of WinUI was made available (10/25/23).  My project file states that the target windows platform is 10.0.19041.0.  I would like to know:

1.  Is my app running WinUI by default when I target windows?
2. How do I find out what version of WinUI my app is running ?
3. How do I update to the latest version of WinUI?

Thanks

**Comment by Xyncgas** [comment]
1. It has to run in WinUI on windows because that's the most modern framework for UI on windows, the MAUI.Blazor project wraps the webview inside WinUI too
2. App Manifest stated a little detail or my guess is do something in the window's platform specific codes in MAUI
3. Updating Visual Studio enables to target the updated version but it's up to the user's computer to provide the WInUI that's why there is a minimal version of WIndows being targeted
